### PR TITLE
[PRODDEV-231] VY Dashboard/Home Reorder Content Feature

### PR DESCRIPTION
**Pull request by NightHunterSV**
**Related Issue/Ticket:** https://openy.atlassian.net/browse/PRODDEV-231

## Steps to test:

- [ ] Log in to a site
- [ ] Go to the app settings (/admin/openy/virtual-ymca)
- [ ] Ensure you have the reordering possibilities:
image

- [ ] Change the order of the components and ensure the changes are applied to the Home, Category, and Favorites pages:
image
image
image

## Quality checks:

Please check these boxes to confirm this PR covers the following cases:

- Maintaining our upgrade path is essential. Check one or the other:
  - [ ] This PR  provides updates via `hook_update_N` or other means.
- Front end fixes should be tested against all of the Open Y Themes.
  - [ ] Tested against Carnation
  - [ ] Tested against Lily
  - [ ] Tested against Rose
- [ ] I have flagged this PR "Needs Review" or pinged the VY devs/QA
  team in Slack

### CODE REVIEW

File: openy_gated_content.libraries.yml [hunk 1]
```diff
@@ -1,3 +1,9 @@
+styles:
```
Comment (hamrant) [inline openy_gated_content.libraries.yml:1]: let's rename lib to `admin-ui-styles`

File: openy_gated_content.libraries.yml [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+  version: 0.1
+  css:
+    theme:
+      assets/css/styles.css: {}
```
Comment (hamrant) [inline openy_gated_content.libraries.yml:5]: let's rename lib to `admin_ui_styles.css`

File: openy_gated_content.libraries.yml (hunk 1 above)
Comment (hamrant) [reply openy_gated_content.libraries.yml:1]: Also, I can't see the place where this lib attached

**Review by hamrant — CHANGES_REQUESTED** [review]

File: openy_gated_content.libraries.yml (hunk 1 above)
Comment (NightHunterSV) [reply openy_gated_content.libraries.yml:1]: It was attached earlier -- just the library declaration was removed someday :)

**Review by retif — APPROVED** [review]
Looks good to me

**Review by hamrant — APPROVED** [review]

**Comment by hamrant** [comment]
Tested, works fine

**Comment by AnastasiiaPys** [comment]
retest

**Comment by AnastasiiaPys** [comment]
Tested, all good!

**Pull request merged**